packages/react-tooltip/src/components/Tooltip/useTooltipStyles.ts

```diff
@@ -0,0 +1,84 @@
+import { makeStyles, ax } from '@fluentui/react-make-styles';
+import { TooltipState } from './Tooltip.types';
+import { Theme } from '@fluentui/react-theme';
+
+/**
+ * The height of the triangle used for the arrow that points at the tooltip's target
+ */
+export const arrowHeight = 6;
+
+export const tooltipBorderRadius = (theme: Theme) => theme.global.borderRadius.medium;
```

Comment:
nit: Is this function necessary or we can just inline value?

Reply:
It's used in useTooltip to for popper's arrowPadding, so that the arrow doesn't appear on top of the rounded corner.